src/measure_theory/group/measure.lean

```diff
@@ -716,6 +716,16 @@ example {E : Type*} [normed_add_comm_group E] [normed_space ℝ E] [nontrivial E
 
 end
 
+variables [nontrivially_normed_field 𝕜] [complete_space 𝕜] [topological_space G] [add_comm_group G]
+  [module 𝕜 G] [finite_dimensional 𝕜 G] [has_continuous_smul 𝕜 G] (μ : measure G)
+  [is_add_haar_measure μ] [borel_space G] [t2_space G] [topological_add_group G]
+  [topological_space H] [add_comm_group H] [module 𝕜 H] [finite_dimensional 𝕜 H]
+  [has_continuous_smul 𝕜 H] [borel_space H] [t2_space H] [topological_add_group H]
+
+instance (e : G ≃ₗ[𝕜] H) : is_add_haar_measure (μ.map e) :=
+e.to_add_equiv.is_add_haar_measure_map _ (e : G →ₗ[𝕜] H).continuous_of_finite_dimensional
+  (e.symm : H →ₗ[𝕜] G).continuous_of_finite_dimensional
```

Comment:
Is finite-dimensionality only used explicitly? If so, could you also add a version where `e : G ≃L[𝕜] H` without finite-dimensionality assumptions?

bors d+

Reply:
Can you also add this to golf the proof of `map_linear_map_add_haar_eq_smul_add_haar`? (where an instance would become unnecessary, as well as a few facts like `Ce` or `Cesymm` only used to prove the instance).

Reply:
I will golf `map_linear_map_add_haar_eq_smul_add_haar` separately because I have more API incoming that will shorten it.

bors merge